src/ingress/views.py

```diff
@@ -1,23 +1,40 @@
 import logging
+from typing import NamedTuple
 
+from django.conf import settings
 from django.http import HttpResponse
 from django.shortcuts import get_object_or_404
 from django.utils.decorators import method_decorator
 from django.views import View
 from django.views.decorators.csrf import csrf_exempt
+from rest_framework.exceptions import AuthenticationFailed
 
 from ingress.models import Endpoint, IngressQueue
 
 logger = logging.getLogger(__name__)
 
 
+class User(NamedTuple):
+    is_authenticated: bool = False
+
+
 @method_decorator(csrf_exempt, name='dispatch')
 class IngressView(View):
     def post(self, request, queue):
+        # Does the called endpoint exist
         endpoint = get_object_or_404(Endpoint, url_key=queue)
         if not endpoint.is_active:
             return HttpResponse("Endpoint is not active anymore", status=404, content_type='text/plain')
 
+        # Is the user authorized to view this endpoint
+        # TODO: replace this with the RBAC authentication
+        if 'HTTP_AUTHORIZATION' not in request.META:
+            return HttpResponse(status=401)
```

Comment:
Eventueel detail message toevoegen:
`HTTP_AUTHORIZATION header not found in request`